Two macOS specific unit tests are failing on linux / windows

### Description
```
  1) window API unit tests privateMethods createWindow hide window until render replicates macOS functionality by creating a fullscreen window from a parent fullscreen window:
     AssertionError: expected { Object (id, webContents, ...) } to have a property '__shouldFullscreen' of true, but got false
      at Function.assert.propertyVal (node_modules/chai/lib/chai/interface/assert.js:1001:37)
      at Context.<anonymous> (test/unit/app/browser/windowsTest.js:215:18)

  2) window API unit tests privateMethods windowRendered makes the window fullscreen if specified:
     AssertionError: expected [Function: proxy] to have a property 'callCount' of 1, but got 0
      at Function.assert.propertyVal (node_modules/chai/lib/chai/interface/assert.js:1001:37)
      at Context.<anonymous> (test/unit/app/browser/windowsTest.js:260:16)
```

### Steps to Reproduce
<!--
Please add a series of steps to reproduce the problem. See https://stackoverflow.com/help/mcve for in depth information on how to create a minimal, complete, and verifiable example.
-->

  1.` npm run unittest`
  2.
  3.


**Actual result:**
fail

**Expected result:**
pass


### Brave Version
master / 0.21 / 0.20

--- a/test/unit/app/browser/windowsTest.js
+++ b/test/unit/app/browser/windowsTest.js
@@ -83,7 +83,7 @@ describe('window API unit tests', function () {
     mockery.registerMock('electron', fakeElectron)
     mockery.registerMock('ad-block', fakeAdBlock)
     mockery.registerMock('../../js/stores/appStore', appStore)
-    mockery.registerMock('../../common/lib/platformUtil', fakePlatformUtil)
+    mockery.registerMock('../common/lib/platformUtil', fakePlatformUtil)
     mockery.registerMock('../../js/dispatcher/appDispatcher', fakeAppDispatcher)
     windows = require('../../../../app/browser/windows')
     appActions = require('../../../../js/actions/appActions')

